plugins/modules/ovirt_host.py

```diff
@@ -625,7 +627,7 @@ def main():
                             search='type=842 or type=841 or type=888',
                         ) if host.name in event.description
                     ]) > 0
-                ),
+                ) or not updates_available),
```

Comment:
The `host` variable does not update over time compared to the `h` in lambda. So when the update would not available the wait_condition will be always true and the timeout.

Reply:
But that is just what we want.
- If there was an update available before the upgrade process started, then you want to wait for event 842/841/888 to make sure host has been rebooted AND is again in the result_state (MAINTENANCE or UP).
- If there was no update available, then we don't have to wait neither.

Reply:
You are right, now we understand the fix, thanks